internal/service/resilience/resilience_external_test.go

```
@@ -69,19 +69,19 @@ func TestBreakerTripsAndRecovers(t *testing.T) {
 	b := res.NewCircuitBreaker(res.BreakerConfig{FailureThreshold: 2, OpenDuration: time.Minute, Clock: clk})
 	fail := func(context.Context) error { return errBoom }
 	//: two failures trip the breaker Open.
-	b.Run(context.Background(), fail)
-	b.Run(context.Background(), fail)
+	b.Run(t.Context(), fail)
```

golangci-lint (2.13.2)

[error] Error return value of `b.Run` is not checked

(errcheck)